front-end/tests/components/LoanDetails.test.js

```diff
@@ -13,7 +13,7 @@ const mockStore = configureStore([thunk])
 describe('<LoanDetails />', () => {
   let component
   let payloads
-  const store = mockStore({ detailLoanId: 1, loanList: data.detailedLoans, loading: true })
+  const store = mockStore({ detailLoanId: 1, loanList: data.detailedLoans, loading: 'DETAIL' })
```

Comment:
@Jeffzholy I know that this was not strictly the part of the change, but if the loading in the state was true or now it has `DETAIL` in it, won't the component only render the loading spinner? so we're just testing that instead of the real component?